(DOCS-3878) Intelligent Test Runner (#15531)
DataDog/documentation: content/en/continuous_integration/troubleshooting.md

If you have admin access, you can update it from the [Repository Settings Page][11].

### Intelligent Test Runner is not working

> [Intelligent Test Runner][12] works by analyzing your commit history along with code coverage information of past test runs to determine which tests need to be run and which ones can be safely skipped. A minimum amount of information need to exist in order for Intelligent Test Runner to work correctly:

Reviewer: ```suggestion
[Intelligent Test Runner][12] works by analyzing your commit history along with code coverage information about past test runs to determine which tests need to be run and which ones can be safely skipped. A minimum amount of information needs to exist in order for the Intelligent Test Runner to work correctly:
```

- Your repository needs to have a commit history of at least 2 commits in the past month.
- You need to have collected test code coverage in past commits.

> Due to these restrictions, the first time you enable Intelligent Test Runner you will not see any tests skipped and the test execution time might be slower than usual due to code coverage being collected automatically.

Reviewer: ```suggestion
Due to these restrictions, the first time you enable Intelligent Test Runner, you cannot see any tests skipped and the test execution time may be slower than usual because the code coverage is collected automatically.
```

> Intelligent Test Runner will only take into account the commit history and test code coverage information for the past month. Additionally, it will not take into account code coverage information that is generated more than one week after a commit was made.

Reviewer: ```suggestion
Intelligent Test Runner only takes into account the commit history and test code coverage information for the past month. Additionally, it does not take into account code coverage information that is generated more than one week after a commit was made.
```
